### DISCUSSION TITLE
"sliders" search term to allow for finding maps with more or less sliders per hit object

### DISCUSSION THREAD

**Discussion by Doksha** [ppy/osu#21399] [Ideas]
would likely be most useful for mania, for finding LN maps, but could also be useful for players who want to find tech maps or farm maps in standard without having to go through each map and check.
The number returned by each map would be <number of sliders> / <number of hitobjects> expressed as a decimal (ie if a map has only sliders/holds it would return 1, and if it only has circles it would return 0, and a 50/50 circles to sliders would return 0.5)
idk if this would be expandable to taiko, probably best just leave out taiko like the "keys" search term.

I know this probably wont be implemented (as its requesting a stat that is not already calculated,) but it would be cool to see.

**Comment by Dopaminos** [comment]
map style in tags: 💀  
(ln, js, hs, dump, tech and etc)

ig your suggestion is based on this RC rule: 
Tags must be related to the beatmap and not misleading for search results. Tags describing the beatmap's style, song, storyboard, video, or background content are considered to be related to the beatmap.

according to this rule and bn.mappersguild.com map style's list, any map should / may have tags mentioning map style, but rn that's optional. so we need RC (please no) or site change, so we could choose map style to search. 

@apollo-dw  any thoughts?

**Reply by Doksha** [reply]
graveyarded maps that may not have tags, as well as just basic ratios for preference, for example you might want something close to 50/50, or maybe 25/75, or anything else. Maybe you want very slider heavy tech, or a tech map with less sliders, finding just the ratio you want makes it a lot easier. Another example would be trying to find maps that would be fun with cs11, you have to find maps with very few sliders or no sliders at all, but since they also have to be very low star they may not be tagged with fitting tags.

**Reply by ghost** [reply]
You're assuming tech maps have more sliders than other mapping styles, which isn't necessarily true. Instead of searching for "sliders" and getting results you might not like or leaving out results you might need, why not just search for "tech" instead, so you can get exactly what you're searching for?

**Reply by Doksha** [reply]
Exactly! I may not be looking specifically for tech, simply wanting more or less sliders. Sorry if I communicated my point wrong, the goal is to be able to search for maps with a large or small amount of sliders, maybe I am looking for a gimmick map, or I simply want more practice with following sliders (maybe I want to practice strict tracking, which isnt very useful if there are few sliders) but I may not be looking for tech.
